### ISSUE TITLE
Improving refutations - simple fix

### ISSUE THREAD

**Issue by aolney** [memphis-iis/mofacts-ies#691]
I'm noticing bad elaborated feedback when the item calls for a non-noun, e.g.`laterally`

My guess is that the ELI5 system we are building on doesn't have this kind of data in its training set. What is needed here is a thesaurus type response rather than response based on conceptual entities.

A simple fix for this would be for me to rebuild the cache and filter out all responses for non-nouns (i.e. non conceptual entities).

Shall I do that?

**Comment by imrryr** [comment]
yes

**Comment by imrryr** [comment]
Added Rusty and Rhodes to handle the handoff

**Comment by aolney** [comment]
Just emailed to @JRustyHaner . 

The new version removes mispelled/nonsensical answers AND removes correct/incorrect pairs that don't have a nominal element on both sides. 

Example nonsensical answers:

- idk
- not sure
- silence detected

Example pairs without a nominal element:

| Incorrect           | Tags  | Correct                | Tags      |
|---------------------|-------|------------------------|-----------|
| lymphatic capillary | JJ JJ | lymphatic vessels      | JJ NNS    |
| lymphatic           | JJ    | thoracic               | NN        |
| digestive           | JJ    | digestion              | NN        |
| unstable            | JJ    | unstable atomic nuclei | JJ JJ NNS |

**Comment by JRustyHaner** [comment]
@aolney @imrryr this has been deployed, closing for now. Thanks @aolney
